modules/environments/service_principal.tf

```diff
@@ -0,0 +1,42 @@
+resource "time_rotating" "one_year" {
```

Comment:
oo this is cool, I didn't know about it, and I see not in ops-bootstrap so new feature!